openslides/assignments/config_variables.py

```diff
@@ -6,15 +6,8 @@
 from openslides.poll.models import PERCENT_BASE_CHOICES
 
 
-def setup_assignment_config(sender, **kwargs):
-    """
-    Receiver function to setup all assignment config variables. They are
-    grouped in 'Ballot and ballot papers' and 'PDF'. This function is
-    connected to the signal openslides.core.signals.config_signal during
-    app loading.
-    """
+def get_config_variables():
```

Comment:
Docstring: `Generator which yields all config variables of this app. They are grouped in 'Ballot and ballot papers' and 'PDF'. The generator has to be evaluated during app loading (see apps.py).`